sdk/script/account-manager.ts

```diff
@@ -45,6 +45,19 @@ interface JsonResponse {
     body?: any;
 }
 
+// A template string tag function that URL encodes the substituted values
+function urlEncode(strings: string[], ...values: string[]): string {
+    var result = "";
+    for (var i = 0; i < strings.length; i++) {
+        result += strings[i];   // Assumes that template string will not start with a substitution
```

Comment:
From what I experimented with template strings,
```javascript
tag`${1}abc${2}${3}`
```
will yield to the tag function 
```
strings: ["", "def", "", ""]
values: [1, 2, 3]
```

I think the spec ensures that the start will be an empty string, and that contiguous substitutions will also be delimited with an empty string. So the assumption is not needed, it is guaranteed by the spec.